charts/lighthouse-webui-plugin/values.yaml

```diff
@@ -25,6 +25,7 @@ secrets:
       secretKeyRef:
         name: lighthouse-hmac-token
         key: hmac
+        optional: true
```

Comment:
I think that instead of making it optional for everybody, you should make it optional only for bitbucket users, by doing similar to https://github.com/jenkins-x/jx3-versions/blob/master/charts/jx3/lighthouse/values.yaml.gotmpl#L60-L63 in https://github.com/jenkins-x/jx3-versions/blob/master/charts/jx3/lighthouse-webui-plugin/values.yaml.gotmpl